src/backend/tests/it/utils/pocketic.rs

```diff
@@ -143,26 +199,38 @@ impl BackendBuilder {
         }
     }
     /// Install the backend canister.
-    fn install(&mut self, pic: &PocketIc) {
+    fn install_backend(&mut self, pic: &PocketIc) {
         let wasm_bytes = self.wasm_bytes();
         let canister_id = self.canister_id(pic);
         let arg = self.arg.clone();
         pic.install_canister(canister_id, wasm_bytes, arg, None);
     }
+    fn install_bitcoin(&mut self, pic: &PocketIc) {
+        let canister_id =
+            Principal::from_text(BITCOIN_CANISTER_ID).expect("Unexpected bitcoin canister id");
+        pic.create_canister_with_id(None, None, canister_id)
+            .expect("Failed creating bitcoin canister");
+        let wasm_bytes = self.bitcoin_wasm_bytes();
+        pic.install_canister(canister_id, wasm_bytes, Self::default_bitcoin_arg(), None);
+    }
```

Comment:
This should technically be in a different builder, as there should be one such builder per canister.  But we can resolve tech debt later.

Reply:
Ditto, I followed the same pattern as the signer canister.

But yes, a different builder makes sense.